MimeKit/Header.cs

```diff
@@ -312,9 +312,14 @@ protected Header (ParserOptions options, HeaderId id, string name, byte[] field,
 		/// <param name="fieldNameLength">The length of the field name (not including trailing whitespace).</param>
 		/// <param name="value">The raw value of the header.</param>
 		/// <param name="invalid"><c>true</c> if the header field is invalid; othereise, <c>false</c>.</param>
+#if NET5_0_OR_GREATER
+		[System.Runtime.CompilerServices.SkipLocalsInit]
+#endif
```

Comment:
Not sure what this does...

Reply:
Ah... https://www.meziantou.net/csharp-9-improve-performance-using-skiplocalsinit.htm

Reply:
Yep. This is what gives us the win. It's basically free to use the stack if we use uncleared memory -- while avoiding the allocation.